bip-0341.mediawiki

```diff
@@ -294,7 +294,15 @@ Examples of preimage for sighashing for each of the sighash modes.
 
 == Deployment ==
 
-TODO
+There may be multiple deployment attempts of this BIP.
+
+The first deployment wil use BIP 8 with the name "taproot", using bit 2, and a lockinontimeout of false.
+
+For Bitcoin mainnet, the BIP 8 startheight will be 681408, the timeoutheight will be 695520, the minimum_activation_height will be 709632, and the activation threshold will be 1815 out of 2016 blocks.
```

Comment:
Perhaps follow "1815 out of 2016 blocks" with "(90 percent)" to make it clear a threshold of 90 percent is set.

Reply:
Done